README.md

```diff
@@ -20,6 +20,9 @@ npm install --save-dev ts-essentials
 👉 We require `typescript>=4.0`. If you're looking for support for older TS versions, please have a look at the
 [TypeScript dependency table](https://github.com/krzkaczor/ts-essentials/tree/master#TypeScript-dependency-table)
 
+👉 As we really want types to be stricter, we require enabled
+[strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) in your project
+
```

Comment:
* I also can leave the link to playground – https://tsplay.dev/mqvpJW (to reproduce the problem)
* Can mention the issue as well – https://github.com/krzkaczor/ts-essentials/issues/196